external/fv3fit/fv3fit/pytorch/cyclegan/generator.py

```diff
@@ -56,22 +63,39 @@ def build(
             kernel_size=self.kernel_size,
             max_filters=self.max_filters,
             convolution=convolution,
+            nx=nx,
+            ny=ny,
+            use_geographic_bias=self.use_geographic_bias,
         )
 
 
+class GeographicBias(nn.Module):
```

Comment:
Assume that later we'd want to have the tile dimension in the bias layer?

Reply:
Oh wow, not just later, but now. This is a bug, will fix.

Reply:
I think this only executes without an error because channels happens to be 1, so it can broadcast over the tile dimension.